packages/ckeditor5-clipboard/tests/dragdropexperimental.js

```diff
@@ -1442,6 +1446,66 @@ describe( 'Drag and Drop experimental', () => {
 				// There should be two elements with the `.ck-content` class - editor and drag-and-drop preview.
 				expect( numberOfCkContentElements ).to.equal( 2 );
 			} );
+
+			it( 'should show preview with custom implementation', () => {
```

Comment:
It's not clear from the test names alone when the editor `should show preview with custom implementation` and when it `should show preview with browser implementation`. Please include information when one or the other is used, so it's not necessary to read the test details.

For example: `should show preview with custom implementation if [...]` and `should show preview with browser implementation if [...]`.